packages/graphback-cli/tests/workflow-test.ts

```diff
@@ -25,30 +24,52 @@ const model = {
   }`
 };
 
-test('Test cli workflow', async (t: ExecutionContext) => {
+test('Test cli workflow', async () => {
   const basePath = resolve(`${__dirname}/../../../templates/ts-apollo-fullstack`);
   process.chdir(basePath);
   await initConfig({ model, database: "sqlite3", client: true, skipInstall: true });
   await generateUsingPlugins({});
 
   await createDB();
 
-  t.true(existsSync(join(basePath, "db.sqlite")));
-  t.true(existsSync(join(basePath, "client/src/graphql/fragments/Note.graphql")));
-  t.true(existsSync(join(basePath, "client/src/graphql/fragments/NoteExpanded.graphql")))
-  t.true(existsSync(join(basePath, "client/src/graphql/fragments/Comment.graphql")))
-  t.true(existsSync(join(basePath, "client/src/graphql/fragments/CommentExpanded.graphql")))
-  t.true(existsSync(join(basePath, "client/src/graphql/queries/findAllNotes.graphql")));
-  t.true(existsSync(join(basePath, "client/src/graphql/queries/findComments.graphql")));
-  t.true(existsSync(join(basePath, "client/src/graphql/queries/findAllComments.graphql")));
-  t.true(existsSync(join(basePath, "client/src/graphql/queries/findNotes.graphql")));
-  t.true(existsSync(join(basePath, "client/src/graphql/mutations/createNote.graphql")));
-  t.true(existsSync(join(basePath, "client/src/graphql/mutations/createComment.graphql")));
-  t.true(existsSync(join(basePath, "client/src/graphql/mutations/deleteComment.graphql")));
-  t.true(existsSync(join(basePath, "client/src/graphql/mutations/deleteNote.graphql")));
-  t.true(existsSync(join(basePath, "client/src/graphql/mutations/updateComment.graphql")));
-  t.true(existsSync(join(basePath, "client/src/graphql/mutations/updateNote.graphql")));
-  t.true(existsSync(join(basePath, "src/schema/schema.graphql")));
-  t.true(existsSync(join(basePath, "src/resolvers/resolvers.ts")));
+  expect(existsSync(join(basePath, "db.sqlite"))).toBe(true);
```

Comment:
Oh.. This must have been so hard. 
We can improve this by snapshotting folder structure.

Reply:
This is the test that didn't really run, so can't confirm if it actually works. Also, that part was done by the jest-codemods.

It was some of the other tests that I had to do manually